src/c++/perf_analyzer/CMakeLists.txt

```diff
@@ -100,4 +102,48 @@ install(CODE "execute_process(COMMAND ${CMAKE_COMMAND} -E create_symlink ./perf_
   WORKING_DIRECTORY ${CMAKE_INSTALL_PREFIX}/bin/)")
 install(CODE "message(\"-- Created symlink: perf_client -> ./perf_analyzer\")")
 
+# Configure doctest
+include(ExternalProject)
+find_package(Git REQUIRED)
+
+ExternalProject_Add(
+    doctest
+    PREFIX ${CMAKE_BINARY_DIR}/doctest
+    GIT_REPOSITORY https://github.com/doctest/doctest.git
```

Comment:
link to a specific tag in the repo. See GIT_TAG.

Reply:
Added 👍